tests/phpunit/tests/rest-api/wpRestTemplatesController.php

```diff
@@ -294,6 +294,26 @@ public function data_get_item_with_valid_theme_dirname() {
 					'post_excerpt' => 'Description of page home template.',
 				),
 			),
+			'template parts: parent theme with non latin characters' => array(
+				'theme_dir' => 'themedir1/block-theme-non-latin',
+				'template'  => 'small-header-%cf%84%ce%b5%cf%83%cf%84',
+				'args'      => array(
+					'post_name'    => 'small-header-τεστ',
+					'post_title'   => 'Small Header τεστ Template',
+					'post_content' => file_get_contents( $theme_root_dir . '/block-theme-non-latin/parts/small-header-τεστ.html' ),
+					'post_excerpt' => 'Description of small header τεστ template.',
+				),
+			),
+			'template: parent theme with non latin name'  => array(
+				'theme_dir' => 'themedir1/block-theme-non-latin',
+				'template'  => 'page-%cf%84%ce%b5%cf%83%cf%84',
+				'args'      => array(
+					'post_name'    => 'page-τεστ',
+					'post_title'   => 'τεστ Page Template',
+					'post_content' => file_get_contents( $theme_root_dir . 'block-theme-non-latin/templates/page-τεστ.html' ),
+					'post_excerpt' => 'Description of page τεστ template.',
+				),
+			),
```

Comment:
For Chinese, it can be done. However, I can't write tests for uppercase. When I try to create a new post with uppercase letters it's transformed to lowercase.
I also tried to create a page with uppercase and even quick-edit a page to add a slug that has uppercase, and always it is transformed to lowercase.

Reply:
@kozer Sounds good. Can you fix up the coding standards issue?

Reply:
@danielbachhuber It's done! Thanks!

Reply:
@danielbachhuber Is there anything else I have to do here? I see you approved the pr, however, I'm not able to merge it.

Reply:
@kozer Did you see this message? https://github.com/WordPress/wordpress-develop/pull/3819#issuecomment-1372349320